test/functionalTest/cases/1048_subscription_cache/three_subscriptions.test

```diff
@@ -261,7 +261,7 @@ Date: REGEX(.*)
 04. Statistics
 ==============
 HTTP/1.1 200 OK
-Content-Length: 222
+Content-Length: 22REGEX(\d)
```

Comment:
Could you elaborate a bit on the 22 -> 22REGEX(\d) change, please? I mean, why this is different depending if tests run under valgrind or not.

Reply:
times < 10 seconds use 1 char, > 10 seconds use 2 chars

Reply:
Maybe it would be more clean just `Content-Length: REGEX(\d*)` but it doesn't matter... it's ok 

NTC